bin/convert-themes-to-rgb.js

```diff
@@ -0,0 +1,25 @@
+const fs = require("fs");
+const path = require("path");
+const tinycolor = require("tinycolor2");
+const colorsWithAlpha = ["toolbar", "toolbar_field"];
+const themesPath = path.join(__dirname, "..", "src", "preset-themes");
+
+fs
+  .readdirSync(themesPath)
+  .filter(filename => path.extname(filename) === ".json")
+  .forEach(filename => {
+    const data = fs.readFileSync(path.join(themesPath, filename), "utf8");
+    const theme = JSON.parse(data);
+    Object.entries(theme.colors).forEach(([name, color]) => {
+      const rgba = tinycolor(color).toRgb();
+      if (!colorsWithAlpha.includes(name)) {
+        delete rgba.a;
+      }
+      theme.colors[name] = rgba;
+    });
+    fs.writeFileSync(
+      path.join(themesPath, filename),
+      JSON.stringify(theme, null, "  "),
```

Comment:
Nit: we'll need to add a `\n` at the end here if we don't want to lose our trailing blank line in those files. 🤷‍♀️

Reply:
Not sure this is important